packages/plugins/plugin-automation/src/components/AutomationContainer.tsx

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import React from 'react';
+
+import { type Space } from '@dxos/react-client/echo';
+import { useTranslation } from '@dxos/react-ui';
+import { ControlSection } from '@dxos/react-ui-form';
+import { StackItem } from '@dxos/react-ui-stack';
+
+import { AutomationPanel } from './AutomationPanel';
+import { AUTOMATION_PLUGIN } from '../meta';
+
+export const AutomationContainer = ({ space }: { space: Space }) => {
+  const { t } = useTranslation(AUTOMATION_PLUGIN);
+  return (
+    <StackItem.Content classNames='block overflow-y-auto'>
+      <div>
```

Comment:
Why this `div`?

Reply:
section doesn't actually wrap its children currently